image/base/slim.Dockerfile

```
@@ -118,4 +118,6 @@ COPY sbin/ /sbin
 # Discourse specific bits
 RUN useradd discourse -s /bin/bash -m -U &&\
     install -dm 0755 -o discourse -g discourse /var/www/discourse &&\
-    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse.git /var/www/discourse
+    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse.git /var/www/discourse &&\
+    cd /var/www/discourse &&\
+    git checkout dev/minio-s3-system-specs
```

I think this should work:

```suggestion
    sudo -u discourse -R /var/www/discourse git checkout dev/minio-s3-system-specs
```

---

Thanks @CvX , was just throwing a hail mary at the end of the day yesterday 😆